main.nf

```diff
@@ -3751,7 +3752,7 @@ process svvcf_to_bed {
 	time '10m'
 
 	when:
-		!params.onco && !params.exome
+		params.antype != "panel"
```

Comment:
sometimes I wonder if there are too many whens. Like in this case this process would not run anyway if the SV-calls were never made.

Reply:
Yes, I agree. I think this would be easier to see if having the code organized into "subworkflows".